setup.py

```diff
@@ -1,9 +1,14 @@
 from setuptools import setup, find_packages
 
 
+# Do not import any validator modules here, as that would require a correct
+# dependency environment that setup.py is not guaranteed to have.
+execfile("validator/_version.py")
+
+
```

Comment:
To check this, we'd have to add another set of tests that didn't install any dependencies. We do have the PyPI deploy script, which should qualify, but that only runs when we tag, and is broken at the moment anyway.

Also, I'm not even sure it's a good idea for `import validator` to work without installing dependencies.